package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "mini-lightbox",
-  "version": "1.3.1",
+  "version": "1.3.2",
   "description": "Minimalist image lightbox",
-  "main": "index.js",
+  "main": "lib/mini-lightbox.js",
```

Comment:
Sure. But that would mean renaming the whole library.
["main" is pointing to the main file to use, when requiring a file.](http://browsenpm.org/package.json#main) In this case you have two files that are of interest.
```
lib/mini-lightbox.js
lib/mini-lightbox.min.js
```

Since I assume that people requiring a file either provide their own minification or don't care about it (node.js), I point to the standard file. If you want to change the naming I recommend splitting the files over the two folders 'lib' and 'dist' too.